charts/rancher-istio/105.5.1+up1.24.1/Chart.yaml

```diff
@@ -11,9 +12,10 @@ annotations:
   catalog.cattle.io/requests-memory: 2314Mi
   catalog.cattle.io/type: cluster-tool
   catalog.cattle.io/ui-component: istio
-  catalog.cattle.io/upstream-version: 1.23.2
+  catalog.cattle.io/upstream-version: 1.24.1
```

Comment:
This is unrelated, right?

Reply:
Yes, it's just updating the upstream version annotation and has no impact on any functionality